### PR TITLE
Add quotes, captions, and misc improvements to Ch6

### PR DESCRIPTION
**Pull request by jni**

### CODE REVIEW

File: markdown/ch6.markdown [hunk 1]
```diff
@@ -825,6 +884,12 @@ what the authors call *detritus* (basically sea sludge), doesn't actually *eat*
 anything (the Circle of Life notwithstanding), so you can't reach other species
 from it.
 
+> *Young Simba:* But, Dad, don't we eat the antelope?
+>
+> *Mufasa:* Yes, Simba, but let me explain. When we die, our bodies become the
+> grass, and the antelope eat the grass. And so we are all connected in the
+> great Circle of Life.
+
```
Comment (hdashnow) [inline markdown/ch6.markdown]: Hehe, cute

**Comment by hdashnow** [comment]
Not sure if you noticed the build error @jni? 

```------------------
idx_lambda3 = np.argmin(np.abs(eigvals - 3))[0]
v3 = Eigvecs[:, idx_lambda3]

print(v3)
print(L @ v3)
------------------

IndexError: invalid index to scalar variable.

Makefile:105: recipe for target 'ipynb/ch6.ipynb' failed
make: *** [ipynb/ch6.ipynb] Error 1```

**Comment by jni** [comment]
I had not, thanks for the ping!

File: markdown/ch6.markdown [hunk 2]
```diff
@@ -183,13 +226,17 @@ import matplotlib.pyplot as plt
...
+Now, we can plot it:
+
 ```python
 import networkx as nx
 g = nx.from_numpy_matrix(A)
 layout = nx.spring_layout(g, pos=nx.circular_layout(g))
 nx.draw(g, pos=layout,
         with_labels=True, node_color='white')
 ```
+<--! caption text="A simple network plotted with NetworkX" -->
```
Comment (stefanv) [inline markdown/ch6.markdown]: Invalid syntax!

**Pull request merged**